modules/vpn/modules/vpn_instance/main.tf

```diff
@@ -0,0 +1,135 @@
+locals {
+  peer_external_gateway = (
+    var.peer_external_gateway != null
+    ? google_compute_external_vpn_gateway.external_gateway[0].self_link
+    : null
+  )
+  secret = random_id.secret.b64_url
+}
+
+resource "google_compute_router" "router" {
+  name    = coalesce(var.router_name, "rtr-${var.name}")
```

Comment:
Maybe we can modify the `var.name` variable into `name_prefix` to be aligned with the other modules/examples ?
Also - the position of this variable here and in subsequent places should be at the beginning of the resource name.

Reply:
Fixed